ktor-server/ktor-server-core/common/test/io/ktor/server/routing/RouteTest.kt

```diff
@@ -122,4 +128,19 @@ class RouteTest {
             assertEquals(expected, endpoints.map { it.toString() }.toSet())
         }
     }
+
+    // KTOR-7449 Ensure request is properly initialized in the routing call
```

Comment:
It's redundant to write the issue ID here because it can be determined from the commit message